src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -162,6 +162,10 @@ public MergeReview getMergeReviewOrThrow(String id) {
 					// Both are deleted, no conflict.
 					persistManualMergeConceptDeletion(mergeReview, conceptId);
 					logger.info("Concept {} deleted on both sides of the merge. Excluding from merge review {}.", conceptId, id);
+				} else if (sourceVersion != null && sourceVersion.isReleased() && targetVersion == null) {
+					// Deleted somewhere, whilst simultaneously versioned elsewhere.
+					persistManualMergeConceptDeletion(mergeReview, conceptId);
+					logger.info("Concept {} versioned somewhere whilst deleted elsewhere. Excluding from merge review {}.", conceptId, id);
```

Comment:
When deleting a Concept and causing a conflict, the UI struggled to display information for the merge screen. 

This will prevent a merge conflict, but will result in the concept later being restored.